django-stubs/template/backends/base.pyi

```diff
@@ -23,6 +21,6 @@ class BaseEngine:
 class _EngineTemplate(Protocol):
     def render(
         self,
-        context: Context | dict[str, Any] | None = ...,
```

Comment:
Why do you remove this? `Context` can be passed to `render()`

Reply:
See the commit comment of 2a070e0a3306875b2d8e436ca4f96083ae36224b: not all implementations of this `render()` method accept `Context` and the spec states that `dict` must be used.

The naming is a bit confusing, as there are multiple classes named `Template` and multiple `render()` methods with different interfaces. The implementer of `_EngineTemplate` is `django.template.backends.django.Template` and the `render()` method of that class doesn't accept `Context`, as it calls `make_context()` which explicitly rejects anything other than `None` and `dict`.

The `django.template.base.Template.render()` method does accept `Context` and in fact requires it, but that is not an implementation of the `_EngineTemplate` protocol, despite using the same class and method name.

Reply:
Ok, I got it! Thanks!